### DISCUSSION TITLE
Display custom parameters in stories.json

### DISCUSSION THREAD

**Discussion by alexandru-sin** [storybookjs/storybook#23965] [Help]
I want to add a parameter to each story
`visuallyTested: true/false`
At the moment these custome parameters are not showing up in stories.json.
Is there a way to list them?
I want to be able to query for these and filter stories.

**Comment by ludvikbrodl** [comment]
I can't figure out how to add parameters. However I found a workaround that gives the same functionallity. Add a tag, with a format that is parsable to include the data your need. For example in our story:

```
const meta: Meta<typeof Button> = {
  component: Button,
  tags: ["autodocs", "visuallyTested: true"],
  parameters: { path: "this does not show up in stories.json, i wish it did though" },
};
```

will generate the output stories.json:
```
...
    "button": {
      ...
      "tags": ["autodocs", "visuallyTested: true"],
      "parameters": {
        "__id": "button",
        "docsOnly": false,
        "fileName": "./stories/Button.stories.tsx"
      }
    },
...
```

and you can use the tags to query/filter instead of parameters. Not a good solution, i'd much rather use Parameters, but it works for me.
